source/geod24/LocalRest.d

```diff
@@ -567,6 +587,12 @@ public final class Timer
             scheduler.schedule(&run);
         }
     }
+
+    /// True if timer is yet to fire
+    @property bool pending () @safe nothrow
+    {
+        return this._pending;
```

Comment:
Maybe and with ‘running’ or setting ‘_pending’  to ‘false’ is required at L558

Reply:
I would go with the second option so the state of `this.pending` is updated as it could be accessed not using this property and have the wrong state.